Gopkg.toml

```diff
@@ -22,7 +22,8 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/common-service"
-  version = "2.2.1"
+#  version = "2.2.1"
+  branch = "ct-2691"
```

Comment:
Instead of doing a commit with a branch would it make sense to create a new release of it ?